packages/wavs/src/subsystems/trigger/streams/atproto_jetstream.rs

```diff
@@ -0,0 +1,784 @@
+        sequence: i64,
+        #[serde(rename = "timeUs")]
+        time_us: i64,
+        repo: String,
+        commit: CommitData,
+    },
+    /// Identity update event
+    Identity {
+        #[serde(rename = "seq")]
+        sequence: i64,
+        #[serde(rename = "timeUs")]
+        time_us: i64,
+        did: String,
+        handle: Option<String>,
+    },
+    /// Account status event
+    Account {
+        #[serde(rename = "seq")]
+        sequence: i64,
+        #[serde(rename = "timeUs")]
+        time_us: i64,
+        did: String,
+        active: bool,
+        status: Option<String>,
+    },
+}
+
+/// Commit data for Jetstream events
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct CommitData {
+    #[serde(rename = "seq")]
+    sequence: i64,
+    rev: String,
+    action: CommitAction,
+    operation: Option<OperationData>,
+    operations: Option<Vec<OperationData>>,
+    #[serde(rename = "ops")]
+    ops: Option<Vec<OperationData>>,
+}
+
+/// Commit action type
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum CommitAction {
+    Create,
+    Update,
+    Delete,
+}
+
+/// Operation data within commits
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct OperationData {
+    path: String,
+    cid: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    action: Option<CommitAction>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    record: Option<Value>,
+}
+
+/// Subscription message for filtering
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct SubscriberMessage {
+    #[serde(rename = "type")]
+    message_type: String,
+    #[serde(rename = "wantedCollections")]
+    wanted_collections: Vec<String>,
+    #[serde(rename = "wantedDids")]
+    wanted_dids: Option<Vec<String>>,
+    cursor: Option<i64>,
+}
+
+/// Parsed ATProto event for internal use
+#[derive(Debug, Clone)]
+pub struct AtProtoEvent {
+    /// Sequence number
+    pub sequence: i64,
+    /// Timestamp in microseconds
+    pub timestamp: i64,
+    /// Repository DID
+    pub repo: String,
+    /// Collection (NSID)
+    pub collection: String,
+    /// Record key
+    pub rkey: String,
+    /// Action type
+    pub action: CommitAction,
+    /// CID of the record
+    pub cid: Option<String>,
+    /// Record data (as JSON)
+    pub record: Option<serde_json::Value>,
+    /// Repository revision identifier for this commit (if provided)
+    pub rev: Option<String>,
+    /// Index of the operation within the commit (0-based)
+    pub op_index: Option<u32>,
+}
+
+/// Create a Jetstream stream for ATProto events
+pub async fn start_jetstream_stream(
+    config: JetstreamConfig,
+    metrics: TriggerMetrics,
+) -> Result<Pin<Box<dyn Stream<Item = Result<StreamTriggers, TriggerError>> + Send>>, TriggerError>
+{
+    let stream = async_stream::stream! {
+        let mut reconnect_count = 0;
+        let max_reconnects = 10;
```

Comment:
As David's comment, same here - this should be configurable via `JetstreamConfig`.